source/sgp_mode/SGPDataNodes.h

```diff
@@ -11,6 +11,10 @@ void SGPWorld::CreateDataFiles() {
       "_SEED" + std::to_string(my_config->SEED()) + ".data";
   // SGP mode doesn't need int val files, and they have significant performance
   // overhead. Only the transmission file needs to be created for this mode.
+  SetUpOrgCountFile(my_config->FILE_PATH() + "OrganismCounts" +
```

Comment:
Should move this down so that the previous comment is still above the correct line